modules/client-configuration/pages/autoinst-pxeboot.adoc

```diff
@@ -0,0 +1,99 @@
+. Prepare an autoinstallation profile, see xref:client-configuration:autoinst-profiles.adoc[Autoinstallation Profiles].
+. Reboot the client, and chose a distribution to install if needed.
+
+Some other steps are optional:
+
+* if you want to use a proxy as the installation server, see xref:client-configuration:autoinst-pxeboot.adoc#synchronize-the-tftp-tree-with-proxies[Synchronize the TFTP Tree with Proxies];
+* if you desire a fully unattended provisioning, see xref:client-configuration:autoinst-xxxxx.adoc[].
+
+
+[[prepare-the-dhcp-server]]
+== Prepare the DHCP Server
+
+The PXE boot process uses DHCP to find the TFTP server. The {productname} Server or its proxies can act as such a TFTP server.
+
+You must have administrative access to the network’s DHCP server. Edit the DHCP configuration file so that it points to the {productname} Server as the TFTP boot server:
+
+.Procedure: Example for Configuring the ISC DHCP Server
+. On the DHCP server, as root, open the [path]``/etc/dhcpd.conf`` file.
+. Modify the declaration for your client following this model:
+----
+  host myclient { (...)
+                  next-server 192.168.2.1;
+                  filename "pxelinux.0"; }
+----
+. Save the file and restart the [systemitem]``dhcpd`` service.
+
+This example:
+
+* directs PXE client "myclient" to the installation server at 192.168.2.1.
+* makes it retrieve the pxelinux.0 bootloader file.
+
+Alternatively, if your DHCP server is registered at {productname}, you can configure it using the DHCPd formula instead:
+
+.Procedure: Example for configuring ISC DHCP Server Using DHCPd Formula
+. Configure the DHCPd formula.
+  For more information, see xref:salt:formula-dhcpd.adoc[DHCPd Formula].
+. In the  [guimenu]``Filename EFI`` field, type [path]``grub/grub.efi`` to enable EFI PXE support.
+. In the  [guimenu]``Filename`` field, type [path]``pxelinux.0`` to enable legacy BIOS support.
+. Click btn:[Save Formula] to save your configuration.
+. Apply the highstate.
+
+
+[[synchronize-the-tftp-tree-with-proxies]]
+== Synchronize the TFTP Tree with Proxies
+
+You can synchronize the TFTP tree on the {productname} Server with the {productname} Proxy.
+For synchronization, HTTPS port 443 must be open.
+
+.Procedure: Synchronizing TFTP between Server and Proxy
+. On the {productname} Server, as root, install the [systemitem]``susemanager-tftpsync`` package:
+----
+zypper install susemanager-tftpsync
+----
+. On the {productname} Proxy, as root, install the [systemitem]``susemanager-tftpsync-recv`` package:
+----
+zypper install susemanager-tftpsync-recv
+----
+. On the {productname} Proxy, as root, run the [command]``configure-tftpsync.sh`` script.
+. The script interactively asks you for details on the host names and IP addresses of the {productname} Server and Proxy, as well for the location of the [path]``tftpboot`` directory on the Proxy. For more information, use the [command]``configure-tftpsync.sh --help`` command.
```

Comment:
Pretty certain I just edited this in a different PR because it was wrong. Probably should check.

Reply:
No clue.